hrm-domain/hrm-core/case/caseSection/caseSectionService.ts

```diff
@@ -84,6 +95,49 @@ export const getCaseSection = async (
   );
 };
 
+export const getCaseTimeline = async (
+  accountSid: string,
+  {
+    user,
+    permissions,
+  }: {
+    user: TwilioUser;
+    permissions: RulesFile;
+  },
+  caseId: number,
+  sectionTypes: string[],
+  includeContacts: boolean,
+  { limit, offset }: ListConfiguration,
+): Promise<TimelineResult> => {
+  return getTimeline(
+    accountSid,
+    user,
+    permissions.viewContact as TKConditionsSets<'contact'>,
+    caseId,
+    sectionTypes,
+    includeContacts,
+    parseInt(limit),
+    parseInt(offset),
```

Comment:
Maybe we could validate the payloads before calling the services? Don't make that change in this PR, I'm just thinking if we want to start doing better payload validation at HTTP requests